libs/ui/design-system/src/lib/checkbox.tsx

```diff
@@ -1,17 +1,32 @@
-import { Checkbox as CheckboxPrimitive } from '@nmit-coursition/ui/primitives/checkbox'
+import * as CheckboxPrimitive from '@nmit-coursition/ui/primitives/checkbox'
 import { Label } from '@nmit-coursition/ui/primitives/label'
+import { cn } from '@nmit-coursition/ui/utils'
+import { type VariantProps, cva } from 'class-variance-authority'
 
-interface CheckboxProps {
+const checkboxVariants = cva('', {
+  variants: {
+    variant: {
+      outlined: '',
+      critical: '', // applies a red border
+    },
+    size: {
+      sm: '', // 8px
```

Comment:
**question:** Where to we have different sizes defined in Figma? We have only one-size of checkbox, right?

Reply:
I actually saw 2 sizes when i inspected, that was why i specified in the comments.
But, I'm going to remove all the unnecessary variants as i already did in other components.